ui/src/app/components/object-explorer/object-explorer.component.scss

```diff
@@ -38,9 +38,12 @@
 .selected-row {
   background-color: #61b3ff4a;
 }
+
 .explorer-table {
   height: calc(100vh - 320px);
   overflow-y: auto;
+  overflow-x: hidden;
```

Comment:
this should not be hidden , scroll bar should not appear if contents do not overflow

Reply:
Updated